src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -568,8 +596,25 @@ def load_arguments(self, _):
                        '--config-file'], help='The config json file.', required=False)
             c.argument('weekday', options_list=[
                        '--weekday'], help='weekday on which maintenance can happen. e.g. Monday', required=False)
-            c.argument('start_hour', type=int, options_list=[
-                       '--start-hour'], help='maintenance start hour of 1 hour window on the weekday. e.g. 1 means 1:00am - 2:00am', required=False)
+            c.argument('start_hour', type=int, required=False, options_list=[
+                       '--start-hour'], help='maintenance start hour of 1 hour window on the weekday. e.g. 1 means 1:00am - 2:00am')
```

Comment:
```suggestion
            c.argument('start_hour', type=int, required=False, options_list=[
                       '--start-hour'], help='maintenance start hour of 1 hour window on the weekday. e.g. 1 means 1:00am - 2:00am')
```
`required=False` is the default setting, please do not explicitly repeat the definition

Reply:
All `required=False` and unwanted `options_list` have been removed